src/ralph/data_center/apps.py

```diff
@@ -0,0 +1,9 @@
+from ralph.apps import RalphAppConfig
+
+
+class DataCenterConfig(RalphAppConfig):
+    name = 'ralph.data_center'
+    verbose_name = 'Data Center Assets'
+
+    def get_proxies(self):
+        return ['DCHost']
```

Comment:
Perhaps a better solution would be to get all of model proxy and exclude class that have a parameter (exclude_permission?)? or classes that have a parameter (permission?)?